### PR TITLE
Update to cosmos v2 for available endpoints

### PR DESCRIPTION
**Pull request by mlunoe**
This PR updates to use the new v2 API endpoints for `/describe` and `/install`

It does not introduce an immediate visual change. Everything should work as before, but the API responses for `/describe` and `/install` changed slightly.

The change is only concerned with Universe actions. Especially pages using `/describe` and `/install`

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @pierlo-upitup

**Comment by mlunoe** [comment]
Retest this please

**Comment by mlunoe** [comment]
Retest this please

**Comment by kennyt** [comment]
px

in master, there's a version number below `kafka`, in this branch there isn't. is that on purpose?

File: src/js/events/CosmosPackagesActions.js [hunk 1]
```diff
@@ -21,11 +21,12 @@ import {
 } from '../constants/ActionTypes';
 import AppDispatcher from './AppDispatcher';
 import Config from '../config/Config';
+import Util from '../utils/Util';
 
 const REQUEST_TIMEOUT = 10000;
 
-function getContentType(action, actionType) {
-  return `application/vnd.dcos.package.${action}-${actionType}+json;charset=utf-8;version=v1`;
+function getContentType(action, actionType, version = 'v1') {
```
Comment (MatApple) [inline src/js/events/CosmosPackagesActions.js]: Should this just default to v2?

Comment (mlunoe) [inline src/js/events/CosmosPackagesActions.js]: Yeah, maybe that is a good idea. I'll change that.
I guess I thought the two `v2`s were the odd ones out, but hopefully the rest will move to v2 soon too

**Comment by kennyt** [comment]
after my one comment, this LGTM

**Comment by MatApple** [comment]
Given my label - just need to address those two comments 👍

**Comment by mlunoe** [comment]
Shows version everywhere now:

**Comment by mlunoe** [comment]
Retest this please

**Comment by mlunoe** [comment]
Retest this please

**Pull request merged**